jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/core/DslTestPlan.java

```diff
@@ -89,6 +95,48 @@ public static DslTestPlan fromJmx(String filePath) throws IOException {
     return new JmxTestPlan(tree);
   }
 
+  /**
+   * if true, thread groups will start consecutively.
+   * One at one time in the order indicated in the plan
+   *
+   * @param serializeTGs enabling or disabling this feature (true/false).
+   * @return this instance for fluent API usage.
+   * @since 0.4
+   */
+  public DslTestPlan setSerialized(Boolean serializeTGs) {
+    this.serializeTGs = serializeTGs;
+    return this;
+  }
+
+  /**
+   * If true, it will cause JMeter to record the data returned from
+   * the server for each sample.
+   * If you have selected a file in your test listeners,
+   * this data will be written to file.
+   *
+   * @param funcMode enabling or disabling this mode (true/false).
+   * @return this instance for fluent API usage.
+   * @since 0.4
+   */
+  public DslTestPlan setGlobalFunctionalMode(Boolean funcMode) {
+    this.funcMode = funcMode;
+    return this;
+  }
+
+  /**
+   * if true, the tearDown groups (if any)
+   * will be run after graceful shutdown of the main threads.
+   * The tearDown threads won't be run if the test is forcibly stopped.
+   *
+   * @param tearDown enabling or disabling this feature (true/false).
+   * @return this instance for fluent API usage.
+   * @since 0.4
+   */
+  public DslTestPlan setTearDownOnShutdown(Boolean tearDown) {
```

Comment:
Take into consideration previous comments on javadoc and method name conventions. Do you see any scenario where it might be worth to set this to false? If not maybe don't make it configurable for the time being, until a need is actually identified?